openstf/pipeline/train_create_forecast_backtest.py

```diff
@@ -18,7 +19,8 @@
 
 
 def train_model_and_forecast_back_test(
```

Comment:
If this function is purely used for integration testing, then we can keep it like this. If this function is used in other pipelines (e.g. icarus forecasts, icarus-openstf-api) we should try to remove modelspecs as input argument to avoid breaking changes